src/components/common/layout/SectionTitle.js

```diff
@@ -20,9 +20,9 @@ const getStylesFromProps = ({ theme }) => {
       fontSize: normalize(24),
       fontWeight: '500',
       textTransform: 'uppercase',
-      marginBottom: normalize(16),
-      marginTop: normalize(16),
-    },
+      marginBottom: normalize(8),
+      marginTop: normalize(8)
```

Comment:
why not use `theme.paddings` or `theme.sizes`... also, you can use `marginVertical` to set those values un a one-liner way.